src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+    <p>Hello, World!</p>
```

Comment:
Tests did not pass, because pages are not identical

Please refer to [this](https://mate-academy.github.io/layout_task-guideline/#check-if-your-solution-matches-all-the-expectations) guide, fix the issues and submit changes

Reply:
@mbruhler

Reply:
@mbruhler 
Corrected, pages are identical.
Passes tests with terminal.
It still shows an error in "Run npm test"
Error: Process completed with exit code 1.